webpack.config.dev.js

```diff
@@ -19,7 +19,11 @@ module.exports = {
     plugins: [
         // new BundleAnalyzerPlugin(),
         new CopyPlugin({
-            patterns: [{ from: 'public/index.html', to: 'index.html' }, { from: 'public/style/*.css' },  { from: 'public/img/endscreen.png', to:'img/endscreen.png' }]
+            patterns: [
+                { from: 'public/index.html', to: 'index.html' },
+                { from: 'public/style/*.css' },
+                { from: 'public/img/exampleGraph.png', to: 'img/exampleGraph.png' }
+            ]
```

Comment:
We need to be able to show images in the game 

_I don't understand how to reference them with webpack so I just did this_